README.md

```diff
@@ -117,3 +130,53 @@
 - [ ] HA of databases and other persistent services.
 
 ## Advance MLonCode awareness
+
+### Make source{d} the face of #MLonCode
+- [ ] Run that workshop at least once at a meetup / conference
+- [ ] Create two demos showing the power of our stack solving a “real problem”
+- [ ] Get at least 6 more collaborators (talk/blog/...) with engineering
+- [ ] Improve the formal process of our internal speaker program in collaboration with Esther and get buyin from Engineering
+
+### Create an "MLonCode friends" program
+- [ ] Define "MLonCode friends" program goals & value proposition
+- [ ] Identify potential members and invite them officially, get 20 to join
+- [ ] Welcome each member personally and provide some swag
+- [ ] Classify members by profile (speakers, researchers, industry/academia, event organizers, etc)
+- [ ] Engage with members and enable them to grow their influence and our message
+
+### Make source{d} a friend of MLonCode academia  
+- [ ] Create a list of all academic events relevant to MLonCode until end of 2019
+  - [ ] Decide level of involvement (attending, presenting, sponsoring, etc)
+- [ ] Release CodeNet with one challenge on top of PGA
```

Comment:
Sorry if that is a stupid question, but what is CodeNet?

Reply:
I am +1 this :smile:

Reply:
CodeNet is just an initial name for the MLonCode equivalent of [ImageNet](http://www.image-net.org/).
A huge dataset with challenges that academia can use as a benchmark for their techniques.